core/src/main/scala/org/apache/spark/eventhubs/client/CachedEventHubsReceiver.scala

```diff
@@ -83,6 +94,7 @@ private[client] class CachedEventHubsReceiver private (ehConf: EventHubsConf,
       logWarning(
         s"$requestSeqNo did not match ${event.getSystemProperties.getSequenceNumber}." +
           s"Recreating receiver for $nAndP")
+      closeReceiver()
```

Comment:
if close fails - we still want to go ahead and create the new Receiver..
So, lets absorb non-fatal errors - and log'em.

Reply:
what change? take a look at the closeReceiver method I added - i log the error there.